includes/alerts/transport.jira.php

```diff
@@ -58,7 +59,10 @@
 $ret  = curl_exec($curl);
 $code = curl_getinfo($curl, CURLINFO_HTTP_CODE);
 if( $code == 200 ) {
-	return true;
+    $jiraout = json_decode($ret, true);
+    log_event("Created jira issue ".$jiraout['key']." for ".$device, $device, 'poller', 5);
```

Comment:
We already record alerts by transport. I don't think we should have individual log_events in here.

You could convert them to d_echo() for debug or we'd need to find a way to allow the return of some data to be used in the central log_event() for alerts.